README.rst

```diff
@@ -73,6 +75,12 @@ For example, we can define the outline for a ``ComplexPhase`` operator in the ``
         (2) of (L, M) sky coordinates with shape (nsrc, 2)
         (3) of frequencies,
       compute the complex phase with shape (nsrc, ntime, nbl, nchan)
+    devices_compatibilities:
+        - "cpu"
+        - "gpu"
+    extra_libraries_deps:
+        - "m"
+    use_old_abi: yes
 
```

Comment:
I think if we start mixing the Makefile options with the operator options, the yaml will start to be cluttered.

How about a `makefile` section to distinguish it from the rest of the yaml?

```yaml
makefile:
    use_old_abi: yes
    libraries:
       - "m"
```